### PR TITLE
Removed react-native-vanilla and react-native-typescript examples.

### PR DESCRIPTION
**Pull request by Gongreg**
## Removed react native vanilla/react native typescript examples.

These examples are not really maintained. Since we are removing packager from storybook and trying to unify experience for crna/rn vanilla, there is no need to keep this code.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -185,11 +185,6 @@ jobs:
       - run:
           name: Bootstrap
           command: yarn bootstrap --reactnative --reactnativeapp
```
Comment (pksunkara) [inline .circleci/config.yml]: `--reactnative` flag should be removed from this line.

File: .teamcity/OpenSourceProjects_Storybook/buildTypes/OpenSourceProjects_Storybook_ReactNative.kt [hunk 2]
```diff
@@ -30,14 +28,6 @@ object OpenSourceProjects_Storybook_ReactNative : BuildType({
...
             dockerImage = "node:%docker.node.version%"
         }
-        script {
-            name = "react-native-vanilla"
-            scriptContent = """
-                cd examples/react-native-vanilla
-                yarn storybook --smoke-test
-            """.trimIndent()
-            dockerImage = "node:%docker.node.version%"
-        }
```
Comment (pksunkara) [inline .teamcity/OpenSourceProjects_Storybook/buildTypes/OpenSourceProjects_Storybook_ReactNative.kt]: There are 2 `--reactnative` flags in this file in other steps. Please correct them.

**Review by pksunkara — CHANGES_REQUESTED** [review]
Also, please remove the `--reactnative` from CONTRIBUTING.md. Otherwise, the PR looks great.

**Comment by Gongreg** [comment]
Thanks @pksunkara, I've updated my PR. :)

**Review by pksunkara — APPROVED** [review]

**Pull request merged**